docs/rootless.md

```diff
@@ -26,7 +26,12 @@ See also the [example manifests](#Kubernetes).
 
 ### Bottlerocket OS
 
-Needs to run `sysctl -w user.max_user_namespaces=N` (N=positive integer, like 63359) on the host nodes.
+Needs to set the max user namespaces to a positive integer, through the [API settings](https://github.com/bottlerocket-os/bottlerocket#kernel-settings):
+
+```toml
+[settings.kernel.sysctl]
+"user.max_user_namespaces" = "16384"
+```
 
 See [`../examples/kubernetes/sysctl-userns.privileged.yaml`](../examples/kubernetes/sysctl-userns.privileged.yaml).
```

Comment:
Probably you want to modify the YAML to update the description about Bottlerocket, and add another YAML for calling the Bottlerocket API (like `sysctl-userns-bottlerocket.privileged.yaml`)

Reply:
Hmm, that yaml file wouldn't be needed since the configurations are set through [user data] and not through a k8s container. Doing so through k8s containers will require additional privileges to the container, which is against Bottlerocket's [security recommendations].

If the file was added specifically for Bottlerocket's use case, I can delete it :+1: 

[user data]: https://github.com/bottlerocket-os/bottlerocket#using-user-data
[security recommendations]: https://github.com/bottlerocket-os/bottlerocket/blob/develop/SECURITY_GUIDANCE.md#restrict-access-to-the-host-api-socket

Reply:
The file can be preserved for other distros